examples/pyinstaller/hook-cefpython3.py

```diff
@@ -95,8 +95,8 @@ def get_excluded_cefpython_modules():
 
 
 def get_cefpython3_datas():
-    """Returning all cefpython binaries as DATAS, because
-    pyinstaller does strange things and fails if these are
+    """Returning almost all of cefpython binaries as DATAS (see exception
+    below), because pyinstaller does strange things and fails if these are
```

Comment:
I have to change it (return subprocess in binary list) because of problem on Darwin. So I decided to be coherent and return executable the same way on all platforms.